firestore-send-email/POSTINSTALL.md

```diff
@@ -47,6 +47,7 @@ The top-level fields of the document supply the email sender and recipient infor
 * **ccUids:** An array containing the CC recipient UIDs.
 * **bcc:** A single recipient email address or an array containing multiple recipient email addresses.
 * **bccUids:** An array containing the BCC recipient UIDs.
+* **headers:** An object of additional header fields (for example, `{"X-Custom-Header": "value", "Y-Second-Custom-Header": "value"}`).
```

Comment:
It'd be best to keep the second header as `"X-Second-Custom-Header"` - Prefixing a header with `X-` is usually practice for non-standard headers.

Reply:
thx! done